server/src/app.js

```diff
@@ -0,0 +1,41 @@
+import nocache from 'nocache';
+import express from 'express';
+import routes from './routes/index.js';
+import handleError from './middlewares/errorHandler.js';
+
+export default class App {
+  express;
```

Comment:
@GritzMaze  Can you explain what is this?

Reply:
This is just a class field. In this case it is public because by default because in JS there are no private fields(if we don't count # way). In typescript this can be private as well.